views/overlays/umsk/game16by9.hbs

```diff
@@ -25,6 +25,28 @@
         document.getElementById('rplayer').innerText = data.rplayer || "";
         document.getElementById('lscore').innerText = data.lscore || "";
         document.getElementById('rscore').innerText = data.rscore || "";
+
+        if(data.lCharacter) {
+          var leftIcon = document.createElement('img');
+          leftIcon.id = 'leftCharacter';
+          leftIcon.className = data.lCharacter;
+          leftIcon.style.zoom = '200%';
+          //src is required to remove the dumb border
+          leftIcon.src = 'http://upload.wikimedia.org/wikipedia/commons/c/ce/Transparent.gif';
+          document.getElementById('lplayer').appendChild(leftIcon);
```

Comment:
As for creating the image in HTML and updating the class, when the text of the names change the image is destroyed and has to be recreated. Logically, it shouldn't be getting destroyed, but this worked. Otherwise I would have done that initially. Any other suggestions?